Classes/Domain/Repository/CompanyRepository.php

```diff
@@ -126,6 +126,10 @@ public function searchCompanies(string $search, int $categoryUid, array $setting
             $constraints[] = $query->logicalOr(...$orConstraints);
         }
 
+        if (isset($settings['district']) && $settings['district'] > 0) {
```

Comment:
In TYPO3 I saw these solutions which are shorter, contains the isset and a convertion to int:
`(int)($settings['district'] ?? 0) > 0`

Reply:
Done